calculation_scripts/L1Norm.py

```diff
@@ -3,16 +3,15 @@
 
 
 posOld = "E:\\Calculations\\2024May10\\HalfCylinder_penalty_parabolic_coeff0.0to0.5_linear\\Structure_Values\\"
-posNew = "E:\\Calculations\\2024May10\\HalfCylinder8_it400_eps_0.01_penalty_parabolic_coeff0.0to0.5_linear\\Structure_Values\\"
+posNew = "E:\\Calculations\\2024May10\\HalfCylinder9_it400_eps0.01_penalty_parabolic_coeff0.0to0.5_linear\\Structure_Values\\"
```

Comment:
done!

Reply:
also FYI: im going to be removing this file from the repo, because its for personal debugging and is not needed for a general user. Trying to keep minimal files in the calculation_scripts folder